cmd/api_serve.go

```diff
@@ -65,6 +65,7 @@ func aptlyAPIServe(cmd *commander.Command, args []string) error {
 	signal.Notify(sigchan, syscall.SIGINT, syscall.SIGTERM)
 	go (func() {
 		if _, ok := <-sigchan; ok {
+			context.TaskList().Wait()
```

Comment:
would be great to add some logging, something like "signal received, waiting for tasks to complete". this would help debugging I think...